.github/ISSUE_TEMPLATE/als-----wil-ik----.md

```diff
@@ -20,14 +20,15 @@ about: Beschrijving van een feature/user story
 - [ ] Is klein genoeg (maximaal 1/5 van sprint)
 - [ ] Product Owner akkoord en voorzien van prioriteit (mag alleen afgevinkt worden door PO)
 - [ ] Idee hebben van hoe deze user story kan worden gedemonstreerd.
-- [ ] _**Userstory is voorzien van een analyse (met daarin architectuur, IM, technische beslissingen afgestemd met team)**_
+- [ ] Userstory is voorzien van een analyse (met daarin architectuur, IM, technische beslissingen afgestemd met team)
 - [ ] Vastgelegd in Github en geplaatst in kolom ready
 
 **Definition of done**
 - [ ] Er is een OAS 3.0 specificatie
 - [ ] Er is een referentieimplementatie
-- [ ] Er zijn tests aanwezig die de wijziging aantonen
+- [ ] Er zijn tests aanwezig die de wijziging aantonen en waarmee de user story getest kan worden.
 - [ ] De technische specificatie (standaard.md) is gepubliceerd leesbaar
+- [ ] Gebruikte informatiemodel is bijgewerkt. (RGBZ/IMZTC zijn inspiratiebron en uitgangspunt maar het door ons 'ontwikkelde' informatiemodel is in lijn met de API-standaarden)
```

Comment:
Wat tussen haakjes staat is meer iets voor de productvisie. Zou ik weglaten.
Het wordt dan "Gegevensmodel is bijgewerkt"